style/row.less

```diff
@@ -34,37 +34,25 @@
   }
 }
 
-.rdg-group-row {
-  > .rdg-cell {
-    // TODO: fix broken border
-    border-right-color: transparent;
-  }
-
-  > .rdg-cell:last-child,
-  > .rdg-cell-frozen-last {
-    border-right-color: @borderColor;
-  }
+.rdg-group-row > .rdg-cell:not(:last-child):not(.rdg-cell-frozen-last) {
+  border-right: none;
 }
 
 .rdg-group-row-selected {
-  .rdg-cell {
-    box-shadow:
-      inset 0 2px 0 0 @selectedBorderColor,
-      inset 0 -2px 0 0 @selectedBorderColor;
-  }
-
-  .rdg-cell:first-child {
-    box-shadow:
-      inset 0 2px 0 0 @selectedBorderColor,
-      inset 0 -2px 0 0 @selectedBorderColor,
-      inset 2px 0 0 0 @selectedBorderColor;
-  }
-
-  .rdg-cell:last-child {
-    box-shadow:
-      inset 0 2px 0 0 @selectedBorderColor,
-      inset 0 -2px 0 0 @selectedBorderColor,
-      inset -2px 0 0 0 @selectedBorderColor;
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    box-shadow: inset 0 0 0 2px @selectedBorderColor;
+    pointer-events: none;
+    z-index: 2;
```

Comment:
I feel this might cause issue for other projects with `z-index`

Reply:
How so? We already have some z-index in rdg.
We also set a stacking context to avoid z-index conflicts.
https://github.com/adazzle/react-data-grid/blob/canary/style/core.less#L4-L5

Reply:
For example if we have a page rendering rdg in the model while a side panel opens which covers the model, potentially there will be some possibilities that this selected border is rendering over the side panel while the model itself is beneath the side panel.

On another thought, maybe we just have to ask them to override the z-index themselves.

Reply:
>there will be some possibilities that this selected border is rendering over the side panel while the model itself is beneath the side panel.

Not possible:
https://github.com/adazzle/react-data-grid/blob/canary/style/core.less#L4-L5
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context

Reply:
nice, missed that comment.